helm_deploy/values-prod.yaml

```diff
@@ -74,6 +74,7 @@ generic-service:
     URL-TEMPLATES_API_CAS3_BOOKING-CONFIRMED-EVENT-DETAIL: https://approved-premises-api.hmpps.service.justice.gov.uk/events/cas3/booking-confirmed/#eventId
     URL-TEMPLATES_API_CAS3_BOOKING-PROVISIONALLY-MADE-EVENT-DETAIL: https://approved-premises-api.hmpps.service.justice.gov.uk/events/cas3/booking-provisionally-made/#eventId
     URL-TEMPLATES_API_CAS3_PERSON-ARRIVED-EVENT-DETAIL: https://approved-premises-api.hmpps.service.justice.gov.uk/events/cas3/person-arrived/#eventId
+    URL-TEMPLATES_API_CAS3_PERSON-ARRIVED-UPDATED-EVENT-DETAIL: https://approved-premises-api.hmpps.service.justice.gov.uk/events/cas3/person-arrived-updated/#eventId
```

Comment:
Should this also show a `URL-TEMPLATES_API_CAS3_PERSON-DEPARTED-UPDATED-EVENT-DETAIL`?

Reply:
No its ` URL-TEMPLATES_API_CAS3_PERSON-DEPARTURE-UPDATED-EVENT-DETAIL`